js/WebView.android.js

```diff
@@ -61,6 +61,7 @@ type State = {|
  */
 class WebView extends React.Component<WebViewSharedProps, State> {
   static defaultProps = {
+    overScrollMode: 'always',
```

Comment:
Maybe `content` makes more sense as default? It's basically 'auto'.

Reply:
Well android default is always, I think we should not alter the current behavior